index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
+    />
+    <link rel="icon" href="/images/favicon.ico" />
+    <link rel="stylesheet" href="css/base.css" />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <div class="nav-content">
+          <div class="logo">
+            <a href="/"
+              ><img src="images/pandamarket.png" width="153" height="51"
+            /></a>
```

Comment:
💊 제안
태그 단위로 개행 해주시는 걸 추천드려요~

```suggestion
            <a href="/">
              <img src="images/pandamarket.png" width="153" height="51"/>
            </a>
```

Reply:
Vscode의 Prettier 에서 자동으로 만들어주어서 어떻게 해야할 지 모르겟습니다! ㅠㅠ